### ISSUE TITLE
Lots of waste of food due to food value and drain

### ISSUE THREAD

**Issue by Dodg-e** [jasonrohrer/OneLifeData7#454]
Current food drain  and food values enables waste of food.

For example eating foods that give a good amount of pips like pies and stew when not hungry or starving especially before adulthood is creating a lot of overeating.

And at the same time the current food drain makes it so eating is a constant thing to do.

Proposed solution:

Dividing by 2 value from every food but also dividing by 2 the current food drain.

So food drain would be slower but the food value would be lower too, which enables the possibility to eat until full and have more time to do other stuff until having to eat again.

Since the food value is divided by 2 there is less waste because overeating is harder

Also this makes food with high value less bad since you dont have to wait until starving to eat them without overeating.

**Comment by Booklat1** [comment]
this isn't really a problem, as overeating is by far less damaging than producing raw crops for food purposes

i`ve demonstrated such here: https://onehouronelife.com/forums/viewtopic.php?id=5047


Honestly Dodge, food data shows more than half the calories we obtain come from berries alone, this is hardly an issue Jason should bother with.

**Comment by trevorhackman** [comment]
Food waste is a problem and more so nobody knows when they're wasting food or how much food they're wasting unless they learn the food values of every food and count out the food bars. 

The hetuw mod does this to the food bar when you hold an edible to show you how much you're filling beforehand:
https://puu.sh/Eiujt/a04ea7b53c.png

I'd suggest something along the lines of that could help a lot.

**Comment by Dodg-e** [comment]
Yes Booklat berries being a main food and giving so much pips that they are still a viable food in late game is an issue too but separate.

To solve that issue berries should give only 2 pips but also eating the same food more than 2-3 times should give "Yuk" and prevent from eating that same food unless starving to prevent berry munching.

Also some food should be in categories to avoid berry bowl being different as berries since they are both berries.

But the current food values is very limiting for gameplay overall, for example adding late game foods that are very nutritious is useless since it will fill you as much as basic foods and increasing their value is useless since the surplus would be waste anyway.

So by lowering the food drain it's possible to also lower food value and make more elaborate foods actually good instead of being a waste of pips and time.

The large berry fields exist because making only more elaborate foods like stew or pies ends up in a waste and being unsustainable in the long run.

Making more elaborate foods takes more time and that makes sense but if half of it is wasted due to overeating then you actually wasted time by making a higher tier item.

A very good example is berry pies, currently it's an absolute waste, but it could be good with different food values but this can only be enabled by changing the food drain

OR by making the food bar bigger, but it already takes a good part of the screen so changing the drain rate is a better option in the long run.

**Comment by Booklat1** [comment]
Dodge, waste in overeating is minimal compared to how wasteful big berry fields are, please do check the post I linked here, it's all about debunking the idea that overreating is a massive waste. It isn't, it has never been. The foods that will usually have wasted pips are incredibly cheaper than berries, to the point it more than makes up for it.  

Despite that we still can survive by having half our food supplies coming from berries. Yum hasnt fixed the issue and so wont yuck because the issue is that they're too cheap/too good. Not in comparison to other foods, but to resources. 


Food doesn't need buffs, if it did we wouldn't be able to survive mostly on berries, we'd die incredibly fast. 


I 100% agree on the main issue that good foods are ridiculously good to the point it doesn't matter. Milk is OP, and so are pies. But then again, in a sense so are berries, which shows how food as a whole is not working incredibly well as a civ survival upkeep cost. Personally I think it would all be fixed if jason removed the serve +2 bonus to foods and added more recipes for early midgame.

**Comment by Booklat1** [comment]
Trevor, read the link I posted please.

There I demonstrate how this waste is minimal compared to other form of waste, like growing berries for food.

In addition, when you compare these with the official food stats you realize people are capable of surviving mostly on berries, with this greater waste. This means the waste from eating pies wrong is hardly harmful longterm.

its worse to eat a berry than to eat a pie at 10 pips, even if you don't overeat the berry. And yet, 50% of all our calories come from berries and that is sufficiently good already.

**Comment by jasonrohrer** [comment]
What you're suggesting, Dodge, is not a clean, isolated fix for food waste.

By slowing food consumption, you increase how long the hunger bar takes to empty, thus changing the rhythm of the game.  Maybe the rhythm needs to be changed, but it shouldn't be changed in order to reduce food waste.


I hear you on the blind trial and error to learn food values.  The client actually doesn't know how much a given food is worth.  It does show how much you get filled, but only by deduction.

Yes, the client has the same data as the server in terms of food value (in the objects folder), but it doesn't know what the current eating bonus is.

The modded client must have the eating bonus hard-coded, which is not correct behavior.

The server could be changed to tell the client what the eating bonus is, I suppose.
